Cargo.toml

```diff
@@ -12,6 +12,7 @@ A minimal deserializer for inspecting `std::backtrace::Backtrace`'s Debug format
 keywords = ["backtrace", "std", "parse"]
 license = "MIT OR Apache-2.0"
 readme = "README.md"
+rust-version = "1.65"
```

Comment:
Added MSRV based on the backtrace feature while I was at it. No strong preference here, I can also drop it again if you'd prefer to not have one.